src/DatabaseWrapper.cpp

```diff
@@ -369,3 +369,43 @@ has_positions(SEXP dbpath)
     Xapian::Database db = Rcpp::as<Xapian::Database>(dbpath);
     return db.has_positions();
 }
+
+//' xapian_delete wrapper
+//' 
+//' @param dbpath path to a Xapian Database
+//' @param docid the document ID of the document to be removed
+//' @examples
+//' \dontrun{
+//' dbpath<- c("path/to/database")
+//' docid <- 20
+//' deleteWrapper(dbpath,docid)
+//' }
+//' 
+//' @return none
+// [[Rcpp::export]]
+void deleteWrapper(SEXP dbpath, SEXP docid){
+  std::string path=Rcpp::as<std::string>(dbpath);
```

Comment:
It'd be better to have indentation consistent across the entire file (previous function uses four spaces).